exist-core/src/main/java/org/exist/xquery/functions/fn/FunCurrentDateTime.java

```diff
@@ -48,11 +48,11 @@ public class FunCurrentDateTime extends Function {
     public final static FunctionSignature fnCurrentDateTime =
         new FunctionSignature(
             new QName("current-dateTime", Function.BUILTIN_FUNCTION_NS),
-            "Returns the xs:dateTimeStamp (with timezone) that is current at some time " +
+            "Returns the xs:dateTime (with timezone) that is current at some time " +
```

Comment:
As with my comment below, this change should be undone.

Reply:
This has also now been undone.